packages/uicore/src/components/MultiSelectDropDown/MultiSelectDropDown.tsx

```diff
@@ -204,20 +206,26 @@ export function MultiSelectDropDown(props: MultiSelectDropDownProps): React.Reac
     const { handleClick, modifiers } = itemProps
     const isSelected = value && value.findIndex(val => val.value === item.value) > -1
     return (
-      <Checkbox
-        key={item.value.toString()}
-        className={cx(css.menuItem, {
-          [css.active]: isSelected,
-          [css.disabled]: modifiers.disabled || item.disabled
-        })}
-        onClick={e => {
-          if (!modifiers.disabled && !item.disabled) {
-            handleClick(e)
+      <div style={{ marginBottom: '5px' }}>
```

Comment:
remove this div. it'll cause style changes for all existing usecases, which we want to avoid